src/store/storeConfig.js

```diff
@@ -2,11 +2,15 @@ import { createStore, combineReducers } from 'redux';
 import exampleReducer from './reducers/exampleReducer';
 import list from './reducers/list';
 
+const persistedState = localStorage.getItem('reduxState') ? JSON.parse(localStorage.getItem('reduxState')) : {}
```

Comment:
this is not redux approach. Got data from `localStorage` are those one, which should go via reducers and passed to state by `combineReducers`.